### ISSUE TITLE
No Default Prefix on list jobs

### ISSUE THREAD

**Issue by dkelosky** [zowe/zowe-cli-ftp-plugin#70]
If I run `zowex jobs list jobs`, I get a response:
C:\>zowex jobs list jobs
```
G0041916 JCL ERROR MULTI    OUTPUT
G0041802 JCL ERROR MULTI    OUTPUT
G0041713 JCL ERROR MULTI    OUTPUT
G0041564 JCL ERROR MULTI    OUTPUT
```

If I run with the `zos-ftp` extension, I get an error for a missing prefix:
```
C:\>zowex zftp list jobs

Syntax Error:
Missing Required Option:
--prefix
```

Can prefix default to `*` - which appears to be what `zowex jobs list jobs` is doing?

**Comment by std4lqi** [comment]
Yes, you can run `zowe zftp ls jobs --prefix "*"` to list all jobs. 

If you know job name, it's more efficient to include `--prefix "G0"` for example.

**Comment by tiantn** [comment]
I think set --prefix "*" as default is reasonable. I will look at it. Thank you @dkelosky !

**Comment by tiantn** [comment]
Hi @dkelosky , this issue was fixed in zftp 1.7.0. Could you help to verify it? Thank you!

**Comment by dkelosky** [comment]
thank you
